Source/TresGame/Public/ETresDropItemID.h

```diff
@@ -5,147 +5,147 @@
-    ITEM_Material02,
-    ITEM_Material03,
-    ITEM_Material04,
-    ITEM_Material05,
-    ITEM_Material06,
-    ITEM_Material07,
-    ITEM_Material08,
-    ITEM_Material09,
-    ITEM_Material10,
-    ITEM_Material11,
-    ITEM_Material12,
-    ITEM_Material13,
-    ITEM_Material14,
-    ITEM_Material15,
-    ITEM_Material16,
-    ITEM_Material17,
-    ITEM_Material18,
-    ITEM_Material19,
-    ITEM_Material20,
-    ITEM_Material21,
-    ITEM_Material22,
-    ITEM_Material23,
-    ITEM_Material24,
-    ITEM_Material25,
-    ITEM_Material26,
-    ITEM_Material27,
-    ITEM_Material28,
-    ITEM_Material29,
-    ITEM_Material30,
-    ITEM_Material31,
-    ITEM_Material32,
-    ITEM_Material33,
-    ITEM_Material34,
-    ITEM_Material35,
-    ITEM_Material36,
-    ITEM_Material37,
-    ITEM_Material38,
-    ITEM_Material39,
-    ITEM_Material40,
-    ITEM_Material41,
-    ITEM_Material42,
-    ITEM_Material43,
-    ITEM_Material44,
-    ITEM_Material45,
-    ITEM_Material46,
-    ITEM_Material47,
-    ITEM_Material48,
-    ITEM_Material49,
-    ITEM_Material50,
-    ITEM_Material51,
-    ITEM_Material52,
-    ITEM_Material53,
-    ITEM_Material54,
-    ITEM_Material55,
-    ITEM_Material56,
-    ITEM_Material57,
-    ITEM_Material58,
-    ITEM_Material59,
-    ITEM_Material60,
-    ITEM_Foodstuff01,
-    ITEM_Foodstuff02,
-    ITEM_Foodstuff03,
-    ITEM_Foodstuff04,
-    ITEM_Foodstuff05,
-    ITEM_Foodstuff06,
-    ITEM_Foodstuff07,
-    ITEM_Foodstuff08,
-    ITEM_Foodstuff09,
-    ITEM_Foodstuff10,
-    ITEM_Foodstuff11,
-    ITEM_Foodstuff12,
-    ITEM_Foodstuff13,
-    ITEM_Foodstuff14,
-    ITEM_Foodstuff15,
-    ITEM_Foodstuff16,
-    ITEM_Foodstuff17,
-    ITEM_Foodstuff18,
-    ITEM_Foodstuff19,
-    ITEM_Foodstuff20,
-    ITEM_Foodstuff21,
-    ITEM_Foodstuff22,
-    ITEM_Foodstuff23,
-    ITEM_Foodstuff24,
-    ITEM_Foodstuff25,
-    ITEM_Foodstuff26,
-    ITEM_Foodstuff27,
-    ITEM_Foodstuff28,
-    ITEM_Foodstuff29,
-    ITEM_Foodstuff30,
-    ITEM_Foodstuff31,
-    ITEM_Foodstuff32,
-    ITEM_Foodstuff33,
-    ITEM_Foodstuff34,
-    ITEM_Foodstuff35,
-    ITEM_Foodstuff36,
-    ITEM_Foodstuff37,
-    ITEM_Foodstuff38,
-    ITEM_Foodstuff39,
-    ITEM_Foodstuff40,
-    ITEM_Foodstuff41,
-    ITEM_Foodstuff42,
-    ITEM_Foodstuff43,
-    ITEM_Foodstuff44,
-    ITEM_Foodstuff45,
-    ITEM_Foodstuff46,
-    ITEM_Foodstuff47,
-    ITEM_Foodstuff48,
-    ITEM_Foodstuff49,
-    ITEM_Foodstuff50,
-    ITEM_Foodstuff51,
-    ITEM_Foodstuff52,
-    ITEM_Foodstuff53,
-    ITEM_Foodstuff54,
-    ITEM_Foodstuff55,
-    ITEM_Foodstuff56,
-    ITEM_Foodstuff57,
-    ITEM_Foodstuff58,
-    ITEM_Foodstuff59,
+    ITEM_HighEther UMETA(DisplayName = "Hi-Ether"),
+    ITEM_Material01 UMETA(DisplayName = "Blazing Shard"),
+	ITEM_Material02 UMETA(DisplayName = "Blazing Stone"),
+	ITEM_Material03 UMETA(DisplayName = "Blazing Gem"),
+	ITEM_Material04 UMETA(DisplayName = "Blazing Crystal"),
+	ITEM_Material05 UMETA(DisplayName = "Frost Shard"),
```

Comment:
Did the formatting change for these? It's showing with extre whitespace on my end.
![image](https://github.com/user-attachments/assets/2a1a6890-2690-4c64-a9c3-d56b91bf7172)

Reply:
The original formatting was for some reason using 4 spaces as indentation, rather than a tabulator. I ended up using a tabulator, which reads as 4 spaces in VS, but 8 here on Github.
Essentially, it makes no difference, but looks kind of odd here.